### DISCUSSION TITLE
초대링크 및 계정설정 문의드립니다.

### DISCUSSION THREAD

**Discussion by jia-shin** [fusiondevkr/pl900#24] [1 Q&A]
안녕하세요. 신지아입니다. 
깃허브 채널에서는 설정한 이름이 jia-shin으로 표시되고, 
팀즈에서는 LNU FNU로 표시되는데
팀즈에서 이름설정이 왜 이렇게 되는지 알 수 있을까요? 

이름이 제대로 표시안되어서 새 계정을 만드려다가 팀초대 수락을 하지 않았습니다. 
혹시 위 사항이 문제되지 않는다면  jia.shin@engineershin.onmicrosoft.com 으로 
초대링크를 한번 더 보내주실수있을까요?
확인 한번 부탁드립니다. 감사합니다. 

<img width="218" alt="스크린샷 2021-07-20 오후 5 51 00" src="https://user-images.githubusercontent.com/87647556/126292238-a63630f0-1248-45eb-86af-4ed67da64c08.png">

 
<img width="278" alt="스크린샷 2021-07-20 오후 5 51 36" src="https://user-images.githubusercontent.com/87647556/126292240-862443bc-a5ec-4b5e-af63-00f631f9c7b9.png">

<img width="960" alt="스크린샷 2021-07-20 오후 5 48 35" src="https://user-images.githubusercontent.com/87647556/126292211-3869dd0d-6ec8-42d1-b741-44f882997430.png">

**Comment by justinyoo** [comment]
@jia-shin 몇가지 가능성이 있습니다.

1. 해당 이메일 주소 jia.shin@engineershin.onmicrosoft.com 를 사용해서 로그인 해 보시면, 계정 이름을 LNU FNU라고 등록하셨을 거예요. 그걸 바꾸시면 됩니다.
2. 이름을 바꾸고 난 후에 다시 한번 https://forms.office.com/r/DuG6xFcn0z 링크를 통해 M365 개발자 계정을 등록해 주세요. 기존 등록 정보는 리셋해 뒀습니다.

**Reply by jia-shin** [reply]
새로운 계정을 하나 더 생성해보았는데, 첫번째 사진처럼 초기값으로 LNU FNU 설정되어있고 jia shin으로 변경했을 때, 깃허브에 표시되는 계정이름만 변경됩니다.  
홈페이지에서는 변경되지 않고, 제가 수정할 수 없다고 나옵니다.


<img width="629" alt="스크린샷 2021-07-21 오후 10 09 19" src="https://user-images.githubusercontent.com/87647556/126515092-21c74842-dd8a-4480-a08a-23f0425a66ed.png">

<img width="320" alt="스크린샷 2021-07-21 오후 11 55 42" src="https://user-images.githubusercontent.com/87647556/126515110-3d4ba701-6b4f-454f-bb48-2a2d0423b6b3.png">

**Reply by justinyoo** [reply]
M365 개발자 계정을 만드시면 자동으로 LNU FNU 혹은 FNU LNU 라고 이름이 붙는데요, 이게 Last Name Unknown, First Name Unknown 의 약자입니다. ㄷㄷㄷ

이거 바꾸시려면 https://admin.microsoft.com/#/users 이리로 들어가셔서 이름 클릭해서 바꾸시면 됩니당.

![image](https://user-images.githubusercontent.com/1538528/126526839-d1dd8cff-80c6-42d7-b1f5-a01171072d43.png)

**Reply by justinyoo** [reply]
@jia-shin 혹시 잘 모르시겠거든 이 페이지를 참조하세요: https://github.com/fusiondevkr/pl900/discussions/27

**Reply by jia-shin** [reply]
계정이름변경이 어디있는지 헤메고 있었는데 연락처 정보관리에 있었네요! 
폼즈 재제출해서 설정 완료했습니다. 감사합니다. 

재 로그인 후, 홈페이지와 팀즈에서의 이름은 초기값에서 jia shin으로 변경된 부분 확인 했습니다.  
그러나 파워 플랫폼 로그인 시, 여전히 FNU LNU's environment라고 표기됩니다. 
혹시, 이 부분은 실습 진행하면서 크게 문제 되지 않는지, 개발 환경이 잘 설정된 것이 맞는지 문의드립니다.

감사합니다! :)

<img width="245" alt="스크린샷 2021-07-23 오전 1 21 21" src="https://user-images.githubusercontent.com/87647556/126674721-60228a45-d886-426e-b9c0-b31966fa82b2.png">

<img width="323" alt="스크린샷 2021-07-23 오전 1 21 00" src="https://user-images.githubusercontent.com/87647556/126674723-3dcfa964-994b-4208-a0f6-7bf62b156d8d.png">

**Reply by justinyoo** [reply]
파워 플랫폼 개발자 계정 생성시 사용했던 이름이 LNU FNU 라면 그걸 바꿀 수는 없습니다. default 환경은 바꿀 수 있지만, 개발자 환경은 못 바꿔요. 그 이름 때문에 사용하는 데 있어서 다른 문제가 있지는 않습니다.

**Reply by jia-shin** [reply]
감사합니다! :)
